src/main/java/service/ReservationService.java

```diff
@@ -22,10 +22,11 @@ public ReservationService(UserService userService, BookService bookService) {
 
     }
 
-    public String createReservation(int id, String nameBook,int isbn) {
-        User user = userService.getUser(id);
-        Book book = bookService.getBook(nameBook,isbn);
-        if (user != null && book != null) {
+    public String createReservation(int userId, String nameBook,String author,int isbn) {
```

Comment:
If you are sending the ISBN, why would you need to send also the name and author?